### PR TITLE
Script to enhance testinterface function in net_data test

### PR DESCRIPTION
**Pull request by hathyaga**
Script includes a function that waits for the interface to be up
and then check for it's status using Ethtool and ip link

Signed-off-by: Harsha Thyagaraja <harshkid@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by hathyaga** [comment]
[net_data_testinterface_PASS.txt](https://github.com/avocado-framework-tests/avocado-misc-tests/files/1047299/net_data_testinterface_PASS.txt)

File: io/net/net_data.py [hunk 1]
```diff
@@ -175,17 +187,12 @@ def testinterface(self):
             self.fail("interface test failed")
         # up the interface
         process.system(if_up, shell=True)
```
Comment (PraveenPenguin) [inline io/net/net_data.py]: as you are sending patch can you please pass` ignore_status=True`

Comment (hathyaga) [inline io/net/net_data.py]: Sure @PraveenPenguin

**Comment by vrbagalkote** [comment]
looks good to me :)

**Comment by PraveenPenguin** [comment]
@vrbagalkote thanks for review

**Pull request merged**